xml/entity-decl.ent

```diff
@@ -125,6 +128,7 @@
 <!ENTITY ais            "OpenAIS">
 <!ENTITY corosync       "Corosync">
 <!ENTITY pace           "Pacemaker">
+<!ENTITY pmrm           "pacemaker_remote">
```

Comment:
Argh, ok. Thanks for the hint!
Hmn, in that case, I should be more precise and add an additional entity:

```xml
<!ENTITY pmrm_daemon    "pacemaker_remoted">
```

Furthermore, I've added a note to make sure the reader is aware of it:

```xml
<note>
   <title>Different Naming for Service and Daemon</title>
   <para>The systemd service of Pacemaker Remote is named &pmremote;
       whereas the name or the process/daemon is &pmrm_daemon;
       (a trailing <quote>d</quote> after its name).
   </para>
</note>
```